src/noos_inv/python.py

```diff
@@ -119,6 +122,8 @@ def release(
         ctx.run(f"poetry publish --build -u {user} -p {token}", pty=True)
     if install_type == InstallType.pipenv:
         raise NotImplementedError
+    if install_type == InstallType.uv:
+        ctx.run(f"uvx twine upload dist/* -u {user} -p {token}")
```

Comment:
**Suggestion:** Ensure that the `uvx` command for publishing uses secure handling of credentials to prevent potential security risks. [security, importance: 9]
```suggestion
        ctx.run(f"uvx twine upload dist/* --username {user} --password {token}", env={"UVX_SECURE_UPLOAD": "1"})
```